content/plugins/psutil.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: bt_wiki
+title: Psutil Plugin
+category: Plugins
+draft: false
+abstract: "Cloudify Psutil plugin description and configuration"
+weight: 1150
+
+---
+{{% gsSummary %}} {{% /gsSummary %}}
+
+
+# Description
+
+The Psutil plugin can be use to collect the metrics from Windows hosts in the similar way the Diamond plugin does it on Linux hosts. It uses [psutil](https://github.com/giampaolo/psutil) underneath and can be configured to run arbitrary psutil functions. The source code is [here](https://github.com/cloudify-cosmo/cloudify-psutil-plugin).
```

Comment:
The Psutil plugin can be use to collect the metrics from Windows hosts in a similar way Diamond plugin does it on Linux hosts. It uses [psutil](https://github.com/giampaolo/psutil) underneath and can be configured to run arbitrary psutil functions. The source code is [here](https://github.com/cloudify-cosmo/cloudify-psutil-plugin).